pkgs/benchmark_harness/README.md

```diff
@@ -114,3 +114,49 @@ Template(RunTime): 0.1568472448997197 us.
 This is the average amount of time it takes to run `run()` 10 times for
 `BenchmarkBase` and once for `AsyncBenchmarkBase`.
 > µs is an abbreviation for microseconds.
+
+## `bench` command
+
+A convenience command available in `package:benchamark_harness`.
+
+If a package depends on `benchmark_harness`, invoke the command by running
```

Comment:
You could also do a `pub global activate`, right? So the two routes are something like
```
dart pub add --dev benchmark_harness
dart run ...
```
or
```
dart pub global activate benchmark_harness
dart pub global run benchmark....